### ISSUE TITLE
Wrong documentation for return type of function gears.color.parse_color()

### ISSUE THREAD

**Issue by JGSVb** [awesomeWM/awesome#3555]
Output of `awesome --version`:
awesome v4.3 (Too long)
 • Compiled against Lua 5.3.6 (running with Lua 5.3)
 • D-Bus support: ✔
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: 0.9.2

**How to reproduce the issue:**
When calling `gears.color.parse_color()` it returns multiple items, not a table, as stated in [gears.color.parse_color()](https://awesomewm.org/doc/api/libraries/gears.color.html#parse_color).

```lua
local t = gears.color.parse_color("#AABBCC") -- not a table, DOES NOT work
```
![image](https://user-images.githubusercontent.com/59124659/151681760-5ae9da02-1f3f-4dd3-8d66-423ecbab0c9d.png)

**Actual result:**
Multiple items are returned instead of a table:

**Expected result:**
It is expected that the function returns a table, so it would be possible to iterate over all color channels (for example).
```lua
-- >>this DOES NOT work<<
local t = gears.color.parse_color("#AABBCC")
-- usage example
for i,v in ipairs(t) do t[i] = t[i] * 2 end
```

**Comment by sclu1034** [comment]
To reduce API breakage, the current return values should not be changed. Only the docs should be aligned to explain that properly, i.e. turn the single `table` return value into four `number` values.

If a table is required, it can always be build by the consumer with `table.pack` or manually writing the literal.

**Comment by Aire-One** [comment]
I think it is resolved now that #3564 is merged.
